.travis.yml

```diff
@@ -0,0 +1,30 @@
+dist: precise
+sudo: false
+language: php
+
+before_script:
+- phpenv config-rm xdebug.ini || return 0
+- composer install
+
+jobs:
+  include:
+  - stage: deploy
```

Comment:
We should also add more staging for testing the code with different php version, like we do in the other plugins (PHP 5.3, 5.4, 5.5 and 7.0 - 7.1 can be ignored as the max PHP version for this plugin is currently 7.0.2)

Reply:
I've taken a look at that. The other plugins either run unit tests or do not run anything. That's because the template for travis was copied, but no one really checked how it works. For example our Framework plugin has this travis:
```
include:
  - stage: Unit Test + Code Style Fixing
    php: 5.3
  - stage: Unit Test + Code Style Fixing
    php: 5.4
  - stage: Unit Test + Code Style Fixing
    php: 5.5
  - stage: Unit Test + Code Style Fixing
    php: 7.0
  - stage: Unit Test + Code Style Fixing
    php: 7.1
  - stage: Unit Test + Code Style Fixing
    php: 5.6
    script:
    - vendor/bin/phpunit
```
On the surface it looks like stuff is happening, but in reality the script tag dictates what is ran for every stage. So what happens here is that vendor/bin/phpunit is ran for every php version. Since it has no config passed-> it runs an empty test call. You can verify by checking the log of one of the php tests for framework:
https://travis-ci.org/shopgate/cart-integration-magento/jobs/305781742

So yea, a lousy job was done migrating our framework and maybe other plugins as well.
So there is nothing to run at the moment for these php versions. The Postman work will start after this is finished as we need to publish. Provided Daniela approves me spending about 2-3 days more on it. On the bright side, I am getting a good understanding of Travis :)

Reply:
For other plugins we are running the tests for different php versions, e. g. https://travis-ci.org/shopgate/cart-integration-prestashop/jobs/305392262 . So yeah, we kind of failed for the old mage plugin.